shell/browser/osr/osr_video_consumer.cc

```diff
@@ -95,12 +95,14 @@ void OffScreenVideoConsumer::OnFrameCaptured(
     texture.coded_size = info->coded_size;
     texture.visible_rect = info->visible_rect;
     texture.content_rect = content_rect;
+    texture.color_space = info->color_space;
     texture.timestamp = info->timestamp.InMicroseconds();
     texture.frame_count = info->metadata.capture_counter.value_or(0);
     texture.capture_update_rect = info->metadata.capture_update_rect;
     texture.source_size = info->metadata.source_size;
     texture.region_capture_rect = info->metadata.region_capture_rect;
     texture.widget_type = view_->GetWidgetType();
+    texture.is_webgpu_compatible = info->metadata.is_webgpu_compatible;
```

Comment:
Are `is_webgpu_compatible` and `supports_zero_copy_webgpu_import` exposed/used anywhere? I see them being copied, but I don't see any other code that references them.